infra/enrolment-records/room-store/src/main/java/com/simprints/infra/enrolment/records/room/store/models/Modality.kt

```diff
@@ -0,0 +1,6 @@
+package com.simprints.infra.enrolment.records.room.store.models
+
+enum class Modality {
```

Comment:
Please add a huge warning not to change the order or remove any lines here since the DB data depends on the ordinal values.

Or use text fields to avoid those issues.